cmd/report_test.go

```diff
@@ -18,32 +18,90 @@ import (
 
 func TestCreateReport(t *testing.T) {
 	t.Parallel()
+
 	logger := testutils.NewLogger(t)
 	opts, err := executor.DeriveScenariosFromShortcuts(lib.Options{
 		VUs:        null.IntFrom(10),
 		Iterations: null.IntFrom(170),
 	}, logger)
 	require.NoError(t, err)
 
-	s, err := execution.NewScheduler(&lib.TestRunState{
-		TestPreInitState: &lib.TestPreInitState{
-			Logger: logger,
-		},
-		Options: opts,
-	}, local.NewController())
-	require.NoError(t, err)
-	s.GetState().ModInitializedVUsCount(6)
-	s.GetState().AddFullIterations(uint64(opts.Iterations.Int64))
-	s.GetState().MarkStarted()
-	time.Sleep(10 * time.Millisecond)
-	s.GetState().MarkEnded()
+	initSchedulerWithEnv := func(lookupEnv func(string) (string, bool)) (*execution.Scheduler, error) {
+		return execution.NewScheduler(&lib.TestRunState{
+			TestPreInitState: &lib.TestPreInitState{
+				Logger:    logger,
+				LookupEnv: lookupEnv,
+			},
+			Options: opts,
+		}, local.NewController())
+	}
 
-	m := createReport(usage.New(), s)
-	require.NoError(t, err)
+	t.Run("default (no env)", func(t *testing.T) {
+		t.Parallel()
+
+		s, err := initSchedulerWithEnv(func(_ string) (val string, ok bool) {
+			return "", false
+		})
+		require.NoError(t, err)
+
+		s.GetState().ModInitializedVUsCount(6)
+		s.GetState().AddFullIterations(uint64(opts.Iterations.Int64))
+		s.GetState().MarkStarted()
+		time.Sleep(10 * time.Millisecond)
+		s.GetState().MarkEnded()
+
+		m := createReport(usage.New(), s)
+		require.NoError(t, err)
+
+		assert.Equal(t, consts.Version, m["k6_version"])
+		assert.EqualValues(t, map[string]int{"shared-iterations": 1}, m["executors"])
+		assert.EqualValues(t, 6, m["vus_max"])
+		assert.EqualValues(t, 170, m["iterations"])
+		assert.NotEqual(t, "0s", m["duration"])
+		assert.EqualValues(t, map[string]any{"ci": false}, m["env"])
```

Comment:
Here and other tests :+1: 
```suggestion
		assert.EqualValues(t, false, m["is_ci"])
```

Reply:
Yeah, yeah, sorry! Forgot to add the changes on the test file, pushing now! Apologies for the noise! 🙏🏻